src/uv-common.h

```diff
@@ -215,4 +211,17 @@ void uv__fs_scandir_cleanup(uv_fs_t* req);
   }                                                                           \
   while (0)
 
+/* Allocator configuration */
+UV_EXTERN extern uv_allocator_t uv__malloc_config;
+
+/* Allocator prototypes */
+void *uv__calloc(size_t count, size_t sz);
+char *uv__strdup(const char *s);
+char *uv__strndup(const char *s, size_t n);
+
+/* Allocator wrappers */
+#define uv__malloc(sz) (uv__malloc_config.local_malloc(sz))
+#define uv__realloc(ptr, sz) (uv__malloc_config.local_realloc((ptr), (sz)))
+#define uv__free(ptr) (uv__malloc_config.local_free(ptr))
```

Comment:
Ah, I see now why you changed `(uv_close_cb) uv__free` above.  I would prefer to keep them as functions.  If nothing else, it's easier when debugging because you can put a breakpoint on them.